src/components/TextInput/TextInput.tsx

```diff
@@ -2,6 +2,7 @@ import React, { forwardRef } from 'react';
 import cx from 'classnames';
 
 type TextInputProps = {
+  autoComplete?: 'name' | 'tel' | 'email';
```

Comment:
Is it worth adding more values here? Looking at MDN, there are many possible values for autocomplete: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete.

I did some testing and I think this works:

```
autoComplete?: HTMLInputElement['autocomplete'];
```

Reply:
Ahh, that only declares it as a string. I was hoping it would be an exhaustive list of all of the values available for `autocomplete`.

Still, maybe better to declare the type as a string? That way we will not need to extend the component if we want to add more values for `autocomplete` in the future.